client/lib/plans/features-list.js

```diff
@@ -990,8 +990,9 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_MEMBERSHIPS ]: {
 		getSlug: () => constants.FEATURE_MEMBERSHIPS,
-		getTitle: () => i18n.translate( 'Sell Subscriptions' ),
-		getDescription: () => i18n.translate( 'Accept monthly or annual payments on your website.' ),
+		getTitle: () => i18n.translate( 'Payments' ),
+		getDescription: () =>
+			i18n.translate( 'Accept one-time, monthly, or annual payments on your website.' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Accept one-time, monthly or annual payments on your website.' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-tweak-payments-wording)